### PR TITLE
PICARD-116: Add restore default button to options

### PR DESCRIPTION
**Pull request by samj1912**

### CODE REVIEW

**Comment by samj1912** [comment]
I tried to make a generic way to load defaults without changing config.setting values but there was a lot of problem with plugin options since their ui elements and config.setting options are not consistently named.

This seemed the other most obvious way.

**Comment by zas** [comment]
The difference between both buttons isn't very obvious.
I think we need a confirmation box for those actions, explaining exactly what will happen when confirmed.

**Comment by samj1912** [comment]
Done :)

File: picard/ui/options/dialog.py [hunk 1]
```diff
@@ -79,11 +79,19 @@ def __init__(self, default_page=None, parent=None):
         self.ui = Ui_Dialog()
         self.ui.setupUi(self)
 
+        self.ui.reset_all_button = QtGui.QPushButton(_("&Restore all Defaults"))
+        self.ui.reset_button = QtGui.QPushButton(_("Restore Defaults"))
```
Comment (zas) [inline picard/ui/options/dialog.py]: Accelerator (`&`) is missing here.

**Comment by samj1912** [comment]
Hadn't added it to Reset Defaults, because of the conflicting Rs :P, added it to D instead.

**Pull request merged**